src/PSGitCompletions/GitRef.cs

```diff
@@ -0,0 +1,28 @@
+﻿using System;
+
+namespace PowerCode
+{
+    public struct GitRef
+    {
+        public enum ObjectType
```

Comment:
In general, Nested types are avoided. See if you can move ObjectType to global scope. Maybe as GitObjectType

Reply:
Moved the enum out of the structure, into an enum `GitObjectType`